databricks/koalas/tests/test_dataframe_conversion.py

```diff
@@ -158,6 +159,8 @@ def test_to_json_with_path(self):
         output_path = "%s/%s" % (self.tmp_dir, output_paths[0])
         self.assertEqual("[%s]" % open(output_path).read().strip(), expected)
 
+    # TODO: `to_clipboard` raises Error since `xclip` seems not work in GitHub Actions on Linux
+    @unittest.skip("Skip for now until `xclip` issue is solved in GitHub Actions")
```

Comment:
Is this because you used `ubuntu-latest`? How does it fail with what kind of errors?